### PR TITLE
Added answer to development plan number 3

### PR DESCRIPTION
**Pull request by jhuseman**
I filled in the answer to number 3 under development plan. Go ahead and make any changes to it you need!

### CODE REVIEW

File: bookfinder/templates/development_plan.html [hunk 1]
```diff
@@ -45,6 +45,13 @@
         </p>
         <h2>Data</h2>
         <p>
+            We will be getting most of our price comparison data for the 
+            application from the internet, from websites like Google Books,
```
Comment (cshinaver) [inline bookfinder/templates/development_plan.html]: I would specify exactly which sites we pull from. E.g. We compare student seller prices against those from Google Books, Barnes & Noble, and Amazon so the buyer can make a market aware decision about buying.

File: bookfinder/templates/development_plan.html [hunk 2]
```diff
@@ -45,6 +45,13 @@
         </p>
         <h2>Data</h2>
         <p>
+            We will be getting most of our price comparison data for the 
+            application from the internet, from websites like Google Books, 
+            Amazon, and Barnes & Noble. Other data about books for sale by 
+            students will obviously come from user input to the application.
```
Comment (cshinaver) [inline bookfinder/templates/development_plan.html]: Little bit too technical. Maybe "Student seller prices will be displayed next to the aforementioned competitors, allowing for a better informed decision about book purchases."

**Comment by jhuseman** [comment]
I corrected the things Charles suggested. I'll be busy for the next few hours, so you'll have to make any more changes and complete the merge yourself.

**Comment by mperez14** [comment]
Approved. Merge it

**Pull request merged**